src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1624,13 +1705,14 @@ protected Void call() {
                     tableViewCandidates.getItems().add(candidate);
                     successCount++;
                   } else {
+                    hasAnyFailures = true;
                     Logger.severe("Failed to load candidate \"%s\"!", name);
                   }
                 }
 
                 Logger.info("Auto-loaded %d candidates.", successCount);
               }
-              return null;
+              return hasAnyFailures;
```

Comment:
Seems like a confusing design to have this one return `true` if `hasAnyErrors`, but all the other instances of `createTask` return `true` if it succeeded?

Reply:
This value is not used anywhere. The callsite uses whether the task itself failed (i.e. an exception was thrown and could not complete), but this return value.

Reply:
Yep, done